libs/application/templates/parental-leave/src/parentalLeaveUtils.ts

```diff
@@ -79,7 +86,44 @@ export function formatPeriods(
 
 /*
  *  Takes in a number (ex: 119000) and
- *  returns a formated ISK value "119.000 kr."
+ *  returns a formatted ISK value "119.000 kr."
  */
 export const formatIsk = (value: number): string =>
   value.toString().replace(/\B(?=(\d{3})+(?!\d))/g, '.') + ' kr.'
+
+/**
+ * Uses `daysInMonth`: 30 as the number of days in a month on average
+ */
+const daysToMonths = (n: number) => n / daysInMonth
+
+/**
+ * Returns the maximum number of months available for the applicant.
+ * Returns as well the days given or requested by the applicant.
+ */
+export const getAvailableRights = (application: Application) => {
+  const requestRights = getValueViaPath(
+    application.answers,
+    'requestRights',
+  ) as any
+  const giveRights = getValueViaPath(application.answers, 'giveRights') as any
+
+  let requestedDays = 0
+  let givenDays = 0
+  let months = defaultMonths
```

Comment:
Just as a reminder: once we connect to paternal leave we'll also need to check existing payment plans to see if applicant has reduced base available rights.

Reply:
Yes, totally. I guess it will be part of the `externalData` so it should be easy to do. I will add a ticket for it